scripts/serverSetup.md

```diff
@@ -0,0 +1,66 @@
+# Getting started with YesWorkflow Web Components
+
+**Please ensure that JDK 10 and Maven 3+ are installed on your machine before continuing and confirm that JDK 10 is your default JAVA_HOME**
+
+## Deploying the web sever
+In the yw_website directory, change directory into the scripts directory. You should see a number of bash scripts:
+```
+deploy_python.sh
+deploy_python3.sh
+remote_deploy_python.sh
+remote_deploy_python3.sh
+workflow_sample.gv
+```
+If you would like to deploy a test server on your localhost, you may run one of these two files:
+```
+deploy_python.sh
+deploy_python3.sh
+```
+For windows users, use deploy_python.sh
+
+For Mac/Linux users, use deploy_python3.sh
+
+These scripts will deploy a web server running on your local machine at 
+```http://localhost:8000```
+
+Deploying on a remote web server is a bit trickier because we will need
+to edit our remote deploy scripts. Let's take a look at ```remote_deploy_python3.sh```
+
+```
+#!/bin/bash
+echo "Copying files to server..."
+scp -r ../../senior-design-group10/ clundin@workflow-web.gonzaga.edu:.
+ssh -t clundin@workflow-web.gonzaga.edu << EOF
+cd ./senior-design-group10/
+pip3 install -r requirements.txt
+cd ./yw_website/
+python3 manage.py makemigrations
+python3 manage.py migrate 
+python3 manage.py makemigrations website
+python3 manage.py migrate website
+python3 manage.py test
+sudo python3 manage.py runserver https://147.222.165.82:80&
+EOF
+```
+
+Here we are ssh-ing into our webserver at 
+workflow-web.gonzaga.edu and running our deploy scripts.
```

Comment:
i like that you are including this, I think it would be a good idea to give two distinct sets instructions:

1. deploying locally to localhost
2. deploying on a dedicated ubuntu server (we should again include what version of the OS we have, not sure off the top of my head)